system/draft/test_misc.py

```diff
@@ -802,7 +802,7 @@ def test_misc_hypothesis_sync(docker_setup_and_teardown, pool_handler, wallet_ha
 async def test_misc_mint_to_aws(payment_init):
     await pool.set_protocol_version(2)
     libsovtoken_payment_method = 'sov'
-    pool_handle, _ = await pool_helper(path_to_genesis='../aws_genesis_test')
+    pool_handle, _ = await pool_helper(path_to_genesis='../aws_genesis')
```

Comment:
`../aws_genesis`? Is it something that should be committed to the repo?

Reply:
I believe that we don't need to run any system tests in CD against AWS pool so no.